heron_wsgi/heron_srv.py

```diff
@@ -770,21 +770,16 @@ def app_factory(global_config, **settings):
 if __name__ == '__main__':  # pragma nocover
     def _script():
         from sys import argv
-
-        # test usage
+        
         from paste import httpserver
-        # from paste import fileapp
+        #using paste to serve /templates/av/ past the proxy 
         host, port = argv[1:3]
 
         logging.basicConfig(level=logging.DEBUG)
 
-        # In production use, static A/V media files would be
-        # served with apache, but for test purposes, we'll use
-        # paste DirectoryApp
-        # TODO: use paster
-        # app = prefix_router('/av/',
-        #                    fileapp.DirectoryApp(HeronAccessPartsApp.htdocs),
-        #                    )
+        app = prefix_router('/av/',
+                            fileapp.DirectoryApp(HeronAccessPartsApp.htdocs),
+                            )
```

Comment:
I don't see any significant changes here. What am I missing?

Did you mean to commit these changes?

"Reviewing the actual diffs as you write the commit message is a good habit to establish. " -- UsingVersionControl

Reply:
oic... you un-commented some stuff.